components/ui/pagination.tsx

```
@@ -0,0 +1,117 @@
+import * as React from "react";
+import { ChevronLeft, ChevronRight, MoreHorizontal } from "lucide-react";
+
+import { cn } from "@/lib/utils";
+import { ButtonProps, buttonVariants } from "@/components/ui/button";
+
+const Pagination = ({ className, ...props }: React.ComponentProps<"nav">) => (
+  <nav
+    role="navigation"
```

Biome (1.9.4)

[error] Using the role attribute 'navigation' on the 'nav' element is redundant, because it is implied by the semantic 'nav' element.

Unsafe fix: Remove the role attribute.


(lint/a11y/noRedundantRoles)